command-expansion-server/src/app.ts

```diff
@@ -27,6 +27,8 @@ app.use(bodyParser.json({ limit: '25mb' }));
 DbExpansion.init();
 const db = DbExpansion.getDb();
 
+const piscina = new Piscina({ filename: path.resolve(__dirname, 'worker.js') });
```

Comment:
__dirname doesn't exist when using ESM. I believe the proper incantation for this use case is
```
new URL('worker.js', import.meta.url).pathname
```